src/main/kotlin/de/achimonline/ansible_lint/settings/AnsibleLintSettings.kt

```diff
@@ -1,6 +1,7 @@
 package de.achimonline.ansible_lint.settings
 
 data class AnsibleLintSettings(
+    var path: String = System.getenv("PATH"),
```

Comment:
As far as I understand this will initialize the "path setting" only once, upon instantiating the data class object.
The current `PATH` value will then be stored and used in every run.
Meaning any updates of the system's `PATH` will never be catched up.